src/scene.ts

```diff
@@ -4123,7 +4123,6 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
     private _processSubCameras(camera: Camera): void {
         if (camera.cameraRigMode === Constants.RIG_MODE_NONE || (camera.outputRenderTarget && camera.outputRenderTarget.getViewCount() > 1 && this.getEngine().getCaps().multiview)) {
             this._renderForCamera(camera);
-            this.onAfterRenderCameraObservable.notifyObservers(camera);

```

Comment:
This is a breaking change, but probably a bearable one as it's only a matter of using `onAfterCameraRenderObservable` instead of `onAfterRenderCameraObservable`.

[EDIT] See comment below, it is in fact a breaking change in WebXR to remove the `onAfterRenderCameraObservable` observable.

Reply:
Rolled back these changes.